t/perf/perf-lib.sh

```diff
@@ -76,6 +76,8 @@ test_perf_copy_repo_contents () {
 		case "$stuff" in
 		*/objects|*/hooks|*/config|*/commondir|*/gitdir|*/worktrees)
 			;;
+		*/fsmonitor)
```

Comment:
I put it on a separate line because it was a different type of thing.  And if we later add other sockets (such as my odb-over-ipc socket), they can go on this line.

Reply:
I don't think it is a different type of thing. As far as this `case` statement is concerned, the question is: should we copy or not copy? And `fsmonitor` is safely in the same category as `objects`, `hooks`, `config`, etc

Reply:
I updated the commit to move it up on the line with the others.